x-pack/plugins/security_solution/public/common/components/events_viewer/events_viewer.tsx

```diff
@@ -323,6 +323,8 @@ const EventsViewerComponent: React.FC<Props> = ({
                       itemsCount: totalCountMinusDeleted,
                       itemsPerPage,
                     })}
+                    leadingControlColumns={[defaultControlColumn]}
```

Comment:
Consider creating constants (outside of the component's closure) for the arrays passed to the `leadingControlColumns` and `trailingControlColumns` props to ensure the same array references are passed as props.